logos/logos-workernode/logos_worker_node/main.py

```
@@ -638,19 +1064,31 @@ def _can_sleep(m: str) -> bool:
     yield
 
     logger.info("Shutting down LogosWorkerNode")
+    # First: stop the re-plan from touching the cache while lanes are being
+    # torn down. A tick mid-shutdown is harmless at worst, but there is no
+    # reason to take "worst" for free.
+    if ram_cache_replan_task is not None:
+        ram_cache_replan_task.cancel()
+        try:
+            await ram_cache_replan_task
+        except (asyncio.CancelledError, Exception):
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)